app/about-about.html

```diff
@@ -3,17 +3,23 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <html>
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; form-action 'none'; referrer no-referrer; script-src 'self'; connect-src 'self'">
-  <meta name="availableLanguages" content="en-US">
-  <meta name="defaultLanguage" content="en-US">
-  <title data-l10n-id="aboutPages"></title>
-  <link rel="localization" href="locales/{locale}/app.l20n">
-  <script src="gen/aboutPages.entry.js" defer></script>
-  <script src="./ext/l20n.min.js" defer></script>
-</head>
-<body>
-  <div id="appContainer"/>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; form-action 'none'; referrer no-referrer; script-src 'self' 'unsafe-inline'; connect-src 'self' ws://localhost:8080">
+    <meta name="availableLanguages" content="en-US">
+    <meta name="defaultLanguage" content="en-US">
+    <title data-l10n-id="aboutPages"></title>
+    <script>
+      window.setTimeout(function() {
+        var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
+        po.src = 'gen/aboutPages.entry.js';
+        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
+      }, 30);
```

Comment:
seems to work with a slight lower delay, but it can't be arbitrarily low because at some point it reverts back to synchronous loading so I kept it at 30 to be safe